### PR TITLE
chore: testing new reusable workflow

### PR DESCRIPTION
**Pull request by Eomm**
Testing the new reusable workflow

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,45 +1,17 @@
...
-    needs: build
-    runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
-      contents: write
-    steps:
-      - uses: fastify/github-action-merge-dependabot@v3
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
+  call-reuseable-workflow:
```
Comment (mcollina) [inline .github/workflows/ci.yml:14]:
```suggestion
  test:
```

**Review by mcollina — COMMENTED** [review]
Seem to work great! Just a note!

**Comment by Eomm** [comment]
There was an error on the coverage step that requires the `--cov --coverage-report=html` args.

```
Run coverallsapp/github-action@master
  with:
    github-token: ***
    parallel: true
    flag-name: run-12-ubuntu-latest
    path-to-lcov: ./coverage/lcov.info
    coveralls-endpoint: https://coveralls.io
Using lcov file: ./coverage/lcov.info
Error: Lcov file not found.
```

Now the error is the following:

```
Run coverallsapp/github-action@master
Using lcov file: ./coverage/lcov.info
Error: Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
```

Not sure the config required to get the coverrals working.

**Comment by mcollina** [comment]
I would just remove coveralls from  everywhere.

**Comment by Fdawgs** [comment]
> 
> Now the error is the following:
> 
> 
> 
> ```
> 
> Run coverallsapp/github-action@master
> 
> Using lcov file: ./coverage/lcov.info
> 
> Error: Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}
> 
> ```
> 
> 
> 
> Not sure the config required to get the coverrals working.
> 
> 

I think I talked about this with @mcollina early last year back when I was trying to standardise the plug-in repos. Some of the repos in the Fastify org do not send coverage to Coveralls or fail to do so if they try (like this one). Fastify-cors is an example of one that does manage to send coverage.

**Comment by mcollina** [comment]
I would just disable it everywhere, it's not worth it (effort vs the information it provides).

**Comment by zekth** [comment]
+1 on disabling coveralls if it's a blocker for this feature

**Comment by Fdawgs** [comment]
@Eomm, v2 of workflows has been released, which has removed all coveralls integration.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,45 +1,15 @@
-name: ci
-'on':
+name: CI
```
Comment (Fdawgs) [inline .github/workflows/ci.yml:1]: The CI badge in the readme may need to be updated if changing from lower to uppercase, but i prefer the uppercase to be honest!

**Review by Fdawgs — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (Eomm) [reply .github/workflows/ci.yml]: Good spot: for consistency in the action menu, I would keep the old `ci` name.

I must say that I like the upper case too 😖

**Comment by Fdawgs** [comment]
LGTM 👍

**Pull request merged**